internal/glfw/glfw_darwin.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build darwin && !js
+// +build darwin,!js
+
+package glfw
+
+import (
+	"github.com/go-gl/glfw/v3.3/glfw"
+)
+
+func init() {
+	glfw.InitHint(glfw.CocoaChdirResources, glfw.False)
```

Comment:
`internal/glfw` should be a pure wrapper. If we need, this should be in `internal/uidriver/glfw`.

Reply:
the CocoaChdirResources behavior is different from other platforms.

Reply:
Again, `internal/glfw` should be a pure wrapper. Don't put this there.